beacon-chain/casper/sharding.go

```diff
@@ -53,21 +52,14 @@ func splitBySlotShard(shuffledValidators []uint32, crosslinkStartShard uint64) [
 }
 
 // getCommitteeParams calculates the parameters for ShuffleValidatorsToCommittees.
-// If numActiveValidators > CycleLength * MinCommitteeSize, multiple committees are selected
-// to attest the same shard in a single slot.
-// If numActiveValidators < CycleLength * MinCommitteeSize, committees span across multiple slots
-// to attest the same shard.
-func getCommitteeParams(numValidators int) (committeesPerSlot, slotsPerCommittee int) {
-	if numValidators >= int(params.GetConfig().CycleLength*params.GetConfig().MinCommiteeSize) {
-		committeesPerSlot := numValidators/int(params.GetConfig().CycleLength*params.GetConfig().MinCommiteeSize*2) + 1
-		return committeesPerSlot, 1
+// If numActiveValidators > CycleLength * MinCommitteeSize, committees are based off a max amount
+// of len(avs) // CYCLE_LENGTH // (MIN_COMMITTEE_SIZE * 2) + 1 or SHARD_COUNT // CYCLE_LENGTH
```

Comment:
Nitpick, can we write out the sentence instead python pseudocode? pseudocode makes it hard to understand

Reply:
Sure thing!